File: server/src/protocol/messages.js
module.exports = {
    createHandshake: (clientId) => ({
        type: "handshake",
        clientId
    }),

    createCommand: (command, data) => ({
        type: "command",
        command,
        data
    }),

    commandsList: {
        moveObject: "moveObject",
        showMessage: "showMessage",
        changeColor: "changeColor",
    },
    messageTypes: {
        handshakeResponse: "handshakeResponse",
        commandCompleted: "commandCompleted"
    }
};

File: unity-client/UnityAnzuProject/Assets/Scripts/Messages.cs
using UnityEngine;

[System.Serializable]
public class Message
{
    public string type;
    public string clientId;
    public string command;
    public CommandData data;
}

[System.Serializable]
public class CommandData
{
    public int x;
    public int y;
    public string text;
    public string color;
}

[System.Serializable]
public class HandshakeResponse
{
    public string type;
    public string clientId;
}

[System.Serializable]
public class CommandCompleted
{
    public string type;
    public string clientId;
    public string command;
    public string result;
}